bindings/core/src/method_handler/secret_manager.rs

```diff
@@ -40,7 +40,7 @@ where
             let addresses = secret_manager
                 .generate_ed25519_addresses(coin_type, account_index, range, options)
                 .await
-                .map_err(iota_sdk::client::Error::from)?
+                .map_err(iota_sdk::client::ClientError::from)?
```

Comment:
Just import iota_sdk::client::ClientError one at the top

Reply:
done